README.md

```diff
@@ -77,10 +84,11 @@ If you are using EvalAI for hosting challenges, please cite the following techni
     volume  =  arXiv:1902.03570
 }
 ```
+The arxiv paper is available [here](https://arxiv.org/abs/1902.03570).
```

Comment:
You might want to put the paper thumbnail here and add href to the arxiv link similar to what we have on our website.